index.html

```diff
@@ -75,29 +75,42 @@ <h6 class="h6">I'm a Software Developer</h6>
     <section class="works">
         <setion class="middlesection">
             <div class="middlesection-container">
-                <h2 class="middlesection_header_title">My Recent Works</h2>
 
-                <hr />
-                <img class="middlesection-image" src="./Assets/images/middlesectionimage.png" alt="" />
-            </div>
-            <h3 class="multiples-stories_header">Multi-Post Stories</h3>
-            <p class="multiples-stories_paragraph">A daily selection of privately personalized reads;
-                no accounts or sign-ups required. has been the industry's standard dummy
-                text ever since the 1500s, when an unknown printer took a standard dummy text.
-            </p>
-            <section class="button-flex">
-                <ul class="buttons">
+                <div class="middlesection_header">
+                    <h2 class="middlesection_header_title">My Recent Works</h2>
+                    <div class="line_container">
+                        <hr>
 
-                    <li><button class="default-btn">CSS</button></li>
-                    <li><button class="default-btn">HTML</button></li>
-                    <li><button class="default-btn">Bootstrap</button></li>
-                    <li><button class="default-btn">Ruby</button></li>
-                </ul>
-            </section>
+                    </div>
+                </div>
+
```

Comment:
Good job working on the desktop version 😃, but you still need to fix some part. Kindly, make sure you replicated the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1278&t=wz325nimR0VV4Kb0-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.) 😄.
### figma

![Capture](https://user-images.githubusercontent.com/57408419/230080511-882df882-36ed-4f43-9de4-29230c3a8001.JPG)

### Your design

![Capture0](https://user-images.githubusercontent.com/57408419/230080575-f12c39a5-5680-4b25-92f0-7fcd8dbb680c.JPG)

### figma 

![Capture2](https://user-images.githubusercontent.com/57408419/230081045-62ef9633-8757-4c51-a280-e0775df9a921.JPG)

### Your design

![Capture1](https://user-images.githubusercontent.com/57408419/230081137-c6b46f0b-f2d9-4608-b14e-154daa5c45f6.JPG)